CHANGELOG.md

```diff
@@ -5,13 +5,84 @@ Releases are named after famous [Cricket](https://en.wikipedia.org/wiki/Cricket)
 
 
 
-## [Unreleased](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/compare/0.7.0..master)
+## [Unreleased](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/compare/0.8.0..master)
 
 ### Breaking changes
 
```

Comment:
```suggestion
- Github organisation OAuth: `auth.github.org_whitelist` has been renamed to `auth.github.orgWhitelist`

```
See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/1118. This is a potential security hole for anyone who doesn't realise this since if no org is whitelisted any GitHub user can login

Reply:
woot! thanks for the suggestion!